input/kafka_go.go

```diff
@@ -50,7 +50,7 @@ func (k *KafkaGo) Init(taskCfg *config.TaskConfig, putFn func(msg model.InputMes
 	kfkCfg := cfg.Kafka[k.taskCfg.Kafka]
 	k.stopped = make(chan struct{})
 	k.putFn = putFn
-	if kfkCfg.Sasl.Enable && kfkCfg.Sasl.Username=="" {
+	if kfkCfg.Sasl.Enable && kfkCfg.Sasl.Username == "" {
```

Comment:
kafka-go support SASL with plain  and scram , maybe we should enable it.

Reply:
`if kfkCfg.Sasl.Enable && kfkCfg.Sasl.Username == "" `   indicates SASL/GSSAPI(KerberosV5)